chart/values.yaml

```diff
@@ -5,6 +5,5 @@ image:
   pullPolicy: IfNotPresent
```

Comment:
Can we get comments in this file pointing to the provider values? I'd be confused as to all the missing defaults here.

Reply:
Done